CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ option(ENABLE_CUDA_MODULES "Enable modules like Random123 written in CUDA" ON)
 option(ENABLE_NET_RECEIVE_BUFFERING "Enable event buffering in net_receive function" ON)
 option(ENABLE_OMP_RUNTIME_SCHEDULE "Use runtime schedule for OpenMP" OFF)
 
+## set C++11 standard to be default
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS ON)
```

Comment:
Could you create an issue about cxx extensions requirement?
(we should fix that in separate PR)

Reply:
I can, but then we'll have to merge that other PR into master and rebase this branch to not get all kinds of build issues

Reply:
ok thats fine